### ISSUE TITLE
Reputation Object and base_score 

### ISSUE THREAD

**Issue by lcostantino** [ocsf/ocsf-schema#1104]
Hi,

as part of https://schema.ocsf.io/1.2.0/objects/reputation?extensions= (and other versions) there's a required field "base_score" that most of the time is not provided.
The most common value i've seen in most product is able to fullfill score_id field.

In this case, that "base_score" is required & float , given it's not provided, what default value should we use?
Shouldn't base_score be OPTIONAL / RECOMMENDED instead?

**Comment by floydtree** [comment]
Agreed, it does make sense to lower the requirement of base_score in the reputation object. And apologies, we couldn't get to this issue earlier. 

@lcostantino feel free to create a PR to reflect this change. Let us know if you need help, #schema channel in the OCSF slack workpsace is good forum any related discussions.
